packages/types/src/component-types.ts

```diff
@@ -577,7 +566,7 @@ export const TEXT_INPUT_PROPS = {
   },
 };
 
-export const NUMBER_INPUT_PROPS = {
+export const TEXT_FIELD_PROPS = {
```

Comment:
If these are only now used in one place (the mappings file for `TextInput` and `TextField` then I'd say let's move them in there to avoid the complicated structure of having to go look them up in a different file.

Otherwise they're fine here.

Reply:
I moved them, but I think I'm maybe missing another reference somewhere